scripts/test-convert.js

```diff
@@ -15,8 +15,9 @@ const appsToConvert = [
   { id: 82250, name: 'search-oauth' },
   { id: 82251, name: 'basic-api-header' },
   { id: 82460, name: 'custom-fields' },
+  { id: 83195, name: 'search-or-create' },
+  { id: 80444, name: 'custom-baisc' },
```

Comment:
`basic` typo, possibly related to a bad merge?

Reply:
Yeah, it came from the merge. Fixed in 6909cc4. Thanks!